cmd/main.go

```diff
@@ -74,18 +74,26 @@ func main() {
 		&config.PluginType, "plugin", "csi-plugin", "Type of this driver i.e. controller or node",
 	)
 
+	cmd.PersistentFlags().StringVar(
+		&configs.QuotaType, "quota-type", "quota", "quota type: refquota or quota",
+	)
+
 	err := cmd.Execute()
 	if err != nil {
 		_, _ = fmt.Fprintf(os.Stderr, "%s", err.Error())
 		os.Exit(1)
 	}
 }
 
-func run(config *config.Config) {
+func run(config *configs.Config) {
 	if config.Version == "" {
 		config.Version = version.Current()
 	}
 
+	if configs.QuotaType != configs.Quota && configs.QuotaType != configs.RefQuota {
+		log.Fatalln(fmt.Errorf("quota-type should be quota or refquota"))
+	}
+
```

Comment:
I think instead of erroring out if that option is not passed it would be better if we set it to `configs.Quota` otherwise this needs to be passed down in upgrade, also kind of changes the default behaviour. cc @tiagolobocastro @niladrih Any thoughts?

Reply:
Additionally, the input option should be either quota or refquota and not any other values?